manuscript/manuscript.Rmd

```diff
@@ -112,11 +111,36 @@ Rich, processes-based simulations such as the SORTIE model in forest management
+These models provide an ideal first benchmark for evaluating the performance of emerging methods of deep RL for several reasons:
+Firstly, for some cases the optimal solution is already known, providing a clear standard-of-comparison to evaluate RL performance. 
+Prior work sometimes overlooks this essential step, assuming that whatever behavior an RL agent produces is sufficiently optimal or otherwise an accurate proxy for behavior arising from animal learning or evolution. 
+As our evaluations will illustrate, such an assumption can be quickly misleading. 
+Second, these models are already widely studied and will be familiar to many readers: @Schaefer1954 is a staple of fisheries management textbooks and practice, with over 2800 citations, while @May1977 has become a canonical model of threshholds and tipping points which still continues to dominate how many ecologists think about these phenomena [@Scheffer2009].  
+Many readers can thus benefit from exisiting knowledge and intutition about the behavior and implications of these models in interpreting the performance of deep RL, something that would not be possible with a more complex model. 
+It would be unwise to consider applications of RL on more complex bespoke models without first analyzing these classic and well-understood cases.
+Third, these models include or can easily be extended to contexts for which the optimal management policy is unknown or inaccessible to classical methods.
+Our implementations of these models in two python modules published to `PyPi` code archive include many such variations which represent open problems for RL.
 We include extensive appendices with carefully annotated code which should allow readers to both reproduce and extend this analysis.
 
+We do recognize the need for more complex, "precise" models to derive quanitatively precise management policy in specific management problems such as fisheries, forestry, or climate change mitigation efforts. 
+The examples here should provide ecologists with the greater background to engage in the collaborative development of such methods, even as they also highlight challenges which ecological problems pose to existing techniques.
+Even the simple models we consider here contain many sources of complexity -- significant stochasticity, non-linear dynamics, hysteresis -- that are likely commonplace to ecological problems but have not been well represtented in RL research to date. 
+Such work is beyond the scope of any one paper: it will require not only more computational resources, but more collaborations between large teams of ecologists and computer scientists.  
```

Comment:
this sentence seems like it should go after the first sentence in the paragraph?